modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -59,6 +72,58 @@ If you provision the adapter with organization credentials, you will be able to
 If you specify environment credentials, you will only be able to manage APIs of that environment. MuleSoft recommends that you use the environment credentials instead of the organization credentials. For more information about which permissions to use, see xref:api-manager::org-credentials-config-mule4.adoc#obtaining-credentials[obtaining credentials].
 * Cluster-admin role access to the Kubernetes cluster.
 
+== Configuring Your Environment Variable
```

Comment:
As mentioned on my previous review, I would only remove the configuration of the path for linux OS.

Reply:
Done! Thanks!